test-project/Assets/EditmodeTests/ModularCodegenTests.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Linq;
+using System.Reflection;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.EditmodeTests
+{
+    [TestFixture]
+    public class ModularCodegenTests
+    {
+        private Assembly generatedCodeAssembly;
+
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            // Assembly.FullName == "<name> <version> <culture> <publickeytoken>", so we just .Contains() it.
+            generatedCodeAssembly = AppDomain.CurrentDomain.GetAssemblies()
+                .First(assembly => assembly.FullName.Contains("Improbable.Gdk.Generated"));
+        }
+
+        // This checks that the codegen module was correctly picked up.
+        [Test]
+        public void Improbable_Gdk_ModularCodegenTests_Test_exists()
+        {
+            Assert.IsTrue(generatedCodeAssembly.GetTypes().Any(type => type.FullName == "Improbable.Gdk.ModularCodegenTests.Test"));
```

Comment:
Why not use `generatedCodeAssembly.GetType("Improbable.Gdk.ModularCodegenTests.Test")` ? It will return null if the type was not found.